### DISCUSSION TITLE
Exclude fields from entry cloning

### DISCUSSION THREAD

**Discussion by ignetic** [ExpressionEngine/ExpressionEngine#3588] [Feature Requests]
The ability to exclude certain fields from being cloned.

The situation may be that it certain fields would be preferred to be left blank when cloning. Some example instances my be where fields need to remain unique use and certain fields containing unique ID's, product SKU's, SEO content. Generally field content you won't want to clone.

It would also be useful to optionally exclude tab content too such as SEO Lite.

Additionally, there could be an option to to set the cloned default status to something other than closed, such as draft.

Alternatively, enable some extension hook where a developer can after the content that can be cloned.

**Comment by intoeetive** [comment]
Pull request: https://github.com/ExpressionEngine/ExpressionEngine/pull/4672
